pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -221,6 +222,11 @@ public Runner build() {
     final TransactionPool transactionPool = pantheonController.getTransactionPool();
     final MiningCoordinator miningCoordinator = pantheonController.getMiningCoordinator();
 
+    AccountWhitelistController accountWhitelistController =
+        new AccountWhitelistController(permissioningConfiguration);
+
+    transactionPool.setAccountWhitelist(accountWhitelistController);
```

Comment:
We don't need to call the setter if the account whitelist property hasn't been set. Why don't we check it here?

```
if (permissioningConfiguration.isAccountWhitelistSet()) {
  transactionPool.setAccountWhitelist(accountWhitelistController);
}
```

Reply:
actioned